app/user/[id]/[postType]/page.tsx

```diff
@@ -19,7 +20,6 @@ export default async function User({ params }: { params: { id: string } }) {
 
     return (
         <>
-            <ProfileHeader user={user} activeUser={isActiveUser} />
```

Comment:
Don't remove the ProfileHeader from here, the ProfileHeader is the main reason for the page to exist and has not to be streamed via @user